### ISSUE TITLE
Option to ignore applying any CSS to Left Toolbar

### ISSUE THREAD

**Issue by apoapostolov** [saif-ellafi/foundryvtt-minimal-ui#28]
I want to use Minimal UI to benefit from specific UI improvements such as the scene thumbnails, the minimal player list, and the tab for minimized windows. However, I wish not to minimize other aspects of the UI.

Unfortunately, the toolbar (the left button column) has CSS applied to it that breaks badly under some skinned systems (PF2 pictured below). I would like to have the option not to skin the toolbar at all, in order to keep the original layout. 

How the original toolbar looks:

![image](https://user-images.githubusercontent.com/4083812/116100690-d8b58c00-a6b5-11eb-990d-ef4338d2593e.png)

How badly Minimal UI works with skinned toolbar at the moment:

![image](https://user-images.githubusercontent.com/4083812/116100161-59c05380-a6b5-11eb-8e75-7cd6c918413d.png)

**Comment by saif-ellafi** [comment]
Good point, going to see what can be done about it.

**Comment by saif-ellafi** [comment]
Hi @apoapostolov can you send me the system/module name to see these skins so I can take as an example?

**Comment by apoapostolov** [comment]
Pathfinder 2 system. Then in the settings tab, open PF2 Player Settings button, and change the skin to In Progress.

**Comment by saif-ellafi** [comment]
Okay, I am looking for ways of improving compatibility.
But for this particular case of pathfinder, you can use the Minimal UI color pickers to find a color that matches the other buttons.
It is Pathfinder the one who does not apply the button styles to those buttons, since those are other modules actually (i.e. Narrator Tools). 
There's also a bit of an issue with the smaller control bar, but if you set the Standard Size, it will be okay.

With just a few trial and error you can achieve this
![image](https://user-images.githubusercontent.com/27952699/117052033-8577b500-ad17-11eb-834c-515e9cc89ce4.png)

In conclusion, I'll soften the color values of minimal ui, so it doesn't interfere much with other styles. But I cannot do much to give another module's buttons the style of another System's UI unfortunately.

Will be closing this issue as it can be a bit misleading. But will continue searching for ways for compatibility.
